scripts/aspnet.benchmarks.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=ASP.NET Benchmarks Server
+
+# Start after network has at least a bound IP address; stop before network is stopped
+After=network.target
+After=network-online.target
```

Comment:
I think we should remove `network-online.target`.  From https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/:

> It is strongly recommended not to pull in this target too liberally: for example network server software should generally not pull this in (since server software generally is happy to accept local connections even before any routable network interface is up), it's primary purpose is network client software that cannot operate without network.